examples/nlp/sentence_embeddings_with_sbert.py

```diff
@@ -0,0 +1,463 @@
+"""
+# Sentence embeddings using siamese BERT-networks
+
+**Author:** [Mohammed Abu El-Nasr](https://github.com/abuelnasr0)<br>
+**Date created:** 2023/05/27<br>
+**Last modified:** 2023/05/27<br>
+**Description:** fine-tune a BERT or RoBERTa model to generate sentence embeddings using
```

Comment:
Fine-tune a BERT or RoBERTa model to generate sentence embeddings using KerasNLP.